larger_than_memory.ipynb

```diff
@@ -0,0 +1,714 @@
+{
```

Comment:
<p>Looks like the examples uses annoy, but since fknni is integrated into ehrapy, we could also use faiss?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>